### ISSUE TITLE
the result of kapao-s

### ISSUE THREAD

**Issue by sayoko17** [wmcnally/kapao#45]
i train the kapao-s，but the result is not well，Is this the difference GPU brings？

YoLov52021-12-14 torch 1.9.1+cu102 CUDA:0(GeForce RTX 2080 SUPER，7982.3125NB)
Average Precision(AP)@[ IoU=0.50:0.95 | area=all | maxDets= 20 ] = 0.587
Average Precision(AP)@[ IoU=0.50| area=all l maxDets= 20 ] = 0.840
Average Precision(AP) [ IoU=0.75i area=alli maxDets= 20 j = 0.644
Average Precision(AP) @[ IoU=0.50:0.95i area=mediumimaxDets= 20 j = 0.542
Average Precision(AP)[ IoU=0.50:0.95i area= large i maxDets= 20 ] = 0.663
Average Recall(AR) @[ IoU=0.50:0.95area=all i maxDets= 20 ] = 0.662
Average Recall(AR) [ IoU=0.50i area=all j maxDets= 20 j = 0.890
Average Recall(AR)@[IoU=0.75i area=alli maxDets= 20 ] = 0.715
Average Recall(AR)@[ IoU=0.50:0.95 area=medium| maxDets= 20 ] = 0.610
Average Recall(AR) IoU=0.50:0.95i area= large | maxDets= 20 ] =0.737.
Speed: 0.518ms pre-process，12.620ms inference，5.793ms ilS per image at shape (1，3，1280，1280)
![wenti](https://user-images.githubusercontent.com/54976956/155506143-3da016f6-06d9-49e6-aaf3-829050a77dfa.png)

and can you tell me what the  keypoint object Fused mean?

**Comment by nikhilchh** [comment]
I have two unrelated questions:
1- What is the size of the model you got for 's' version. I get 101.2 mb model and I was expecting 25.7 mb model. 
2- Are the '2307 missing data in validation data' images which don't have any human label ?

**Comment by wmcnally** [comment]
@sayoko17 Yes, I suspect this could be a result of lower batch size. Can you share your training command? Keypoints objects fused is the number of keypoint objects that were fused with pose objects during validation. It only prints now with the `--count-fused` option, so it looks like your code is a few commits behind.

@nikhilchh 
1. It should be around 25.7 MB but it could be much higher during training. The optimizer weights are stripped at the end of training which reduces the file size.
2. Yes.

**Comment by nikhilchh** [comment]
@wmcnally 
If I start training for 100 epochs. And after 100 epochs optimizer weights are stripped. 
Does it mean I cannot resume training from there ?

**Comment by wmcnally** [comment]
If you do that you cannot use the `--resume` argument but you can initialize a new model with your pretrained weights using the `--weights` argument.

**Comment by wmcnally** [comment]
I don't recall exactly.. But I think 2-3 days using a batch size of 256.

**Comment by namehcs** [comment]
@wmcnally  I am trying to deploy this model on embedded AI devices. Due to the long training cycle of the model, I plan to change the pre-training model of Yolov5s6 to Yolov5s. Have you tried using a smaller backbone network before?

**Comment by wmcnally** [comment]
Yes, I have. That's what I started with. If I recall correctly the accuracy is approximately ~2 AP lower but it trains much faster. You should use the following arguments: `--weights yolov5s.pt --imgsz 640 --hyp data/hyps/hyp.kp.yaml`. If training time is still an issue, you can use `--noval` for a small speedup. Hope this helps.
